source/agora/test/BanManager.d

```diff
@@ -79,42 +80,42 @@ unittest
     // leftover txs which node 3 will reject due to its filter
     Transaction[] left_txs;
 
-    auto new_tx = genBlockTransactions(4);
-    left_txs ~= new_tx;
-    new_tx.each!(tx => node_1.putTransaction(tx));
+    foreach (block_idx; 0 .. 4)
+    {
+        auto new_tx = genBlockTransactions(1);
+        left_txs ~= new_tx;
+        new_tx.each!(tx => node_1.putTransaction(tx));
+
+        [node_1, node_2].enumerate.each!((idx, node) =>
+            retryFor(node.getBlockHeight() == 1 + (block_idx + 1),
+                4.seconds,
+                format("Block %s Node %s has block height %s. Expected: %s",
+                    block_idx + 1, idx, node.getBlockHeight().to!string, 1 + (block_idx + 1))));
```

Comment:
Why the `to!string` here ?

Reply:
It used to be in place of the `format` call, I'll fix it.